.github/workflows/lint.yml

```diff
@@ -34,9 +34,12 @@ jobs:
         with:
           python-version: ${{ matrix.python-version }}
 
-      - name: Install dependencies
+      - name: Install numpy (needed for cpp script compilation)
         run: |
           python -m pip install --upgrade pip
+          python -m pip install numpy
+          python setup.py build_ext --inplace
+          python -c "import sys; print(sys.path)"
```

Comment:
In addition to the following line, I had to install torch with:
```
pip install torch
```

Reply:
Uh? `torch` is in the `requirements.txt` list, it should be installed through `pip install -r requirements-dev.txt`.

Anyway, the PR is still buggy, as there is a Python packaging issue in the CI...